discovery/targetgroup/targetgroup.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/prometheus/common/model"
 )
 
-// Group is a set of targets with a common label set(production , test, staging etc.).
+// Group is a set of targets with a common label set(production, test, staging etc).
```

Comment:
Change seems unrelated.

Reply:
Yes, unrelated but entirely routine and uncontroversial. Are you suggesting I remove it?

Reply:
Yes.  When I look back over the history of a file I want to see changes labeled with the reason they were made.
For me it is routine and uncontroversial to put distinct changes in different PRs.

Reply:
Okay, fair enough. Will also remove the new lines I added elsewhere in the PR.